bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/Radion_VBF_WW_inclu/Radion_VBF_WW_inclu_narrow_M/Radion_VBF_WW_inclu_narrow_M_customizecards.dat~

```diff
@@ -0,0 +1,3 @@
+set param_card mass 35 800
```

Comment:
please remove .dat~ files

Reply:
yes I realized it too late, I am working on it, sorry